src/core/cgroup.h

```diff
@@ -153,8 +152,11 @@ struct CGroupContext {
         bool memory_min_set;
         bool memory_low_set;
 
-        LIST_HEAD(IPAddressAccessItem, ip_address_allow);
-        LIST_HEAD(IPAddressAccessItem, ip_address_deny);
+        Set *ip_address_allow;
+        Set *ip_address_deny;
+        /* in_addr_prefixes_reduce() is called. */
+        bool ip_address_allow_reduced;
+        bool ip_address_deny_reduced;
```

Comment:
`:1` on bot?

Reply:
Unfortunately, it cannot. In bpf-firewall.c, pointers to them are used.